### DISCUSSION TITLE
Copy to LBP3

### DISCUSSION THREAD

**Discussion by RedRavenYouTube** [LBPUnion/ProjectLighthouse#606] [Ideas]
I don't have code but can you make it so Union will check if you have an LBP3 PS4 profile and if you do can you make it duplicate the level and auto-publish to the LBP3 PS4 server?

**Comment by indo2042** [comment]
this is an awesome idea but that’s just not possible as these are two completely different servers

**Comment by m88youngling** [comment]
> I don't have code but can you make it so Union will check if you have an LBP3 PS4 profile and if you do can you make it duplicate the level and auto-publish to the LBP3 PS4 server?

This is impossible as there is no way to automatically access the LBP PS4 server. The endpoints are locked down. If this was possible, the LBP PS4 servers would have been hacked by now lol

**Comment by RedRavenYouTube** [comment]
Well you could if you teamed up with Sumo but that probably is never likely to happen because when the servers were hacked and the bad messages popped up, it was the community and since you are part of said community they probably don't trust you.

**Reply by m88youngling** [reply]
Yeah, no. No entity of Sony will ever partner with this project lol
